frontend/src/views/web/partials/sidebar.html

```diff
@@ -14,15 +14,15 @@
                 </li>
                 <li>
                     <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-users side-link-icon"></i> Teams</a>
+                        <i class="fa fa-users side-link-icon"></i> My Participant Teams</a>
```

Comment:
So currently it is a view to choose the team for a particular challenge. We have to make it independent where user can see the list of teams or just create a team. 

When user will try to participate in a challenge, then he will be shown a similar page from where he can select one team from list of teams or create a new team and then proceed.